src/main.ts

```diff
@@ -46,12 +47,26 @@ export async function createFileStructure(command: Command) {
       return ReturnCode.InternalError
     })
 }
-export async function generateDocs(command: Command) {
+export async function doc(command: Command) {
+  const subCommand = command.getSubCommand()
+
+  if (subCommand === 'init') {
+    return await initDocs()
+      .then(() => {
+        displaySuccess(`Docs are initialited. You can generate docs now.`)
```

Comment:
_initialised_

Reply:
The doxygen configuration files have been initialised under `/sasjs/doxy/`.  You can now run `sasjs doc`.